snippets/curl/get_nb_assets.sh

```diff
@@ -0,0 +1,116 @@
+!/bin/sh
+
+#####################n#####################################################
+
+# This script demonstrates the usage of netbackup REST API for listing
+# the assets
+
+# This script requires jq command-line JSON parser
+# if your system does not have jq installed, this will not work
+# jq can be downloaded from here: https://github.com/stedolan/jq/releases
+
+###########################################################################
+
+port=1556
+master_server=""
+username=""
+password=""
+domainname=""
+domaintype=""
+
+showHelp()
+{
+        echo ""
+        echo "Invalid command parameters"
+        echo "Usage:"
+        echo "./get_nb_vmservers.sh -nbmaster <master_server> -username <username> -password <password> -domainname <dname> -domaintype <unixpwd/nt>"
```

Comment:
Command doesn't match the file name.

```suggestion
        echo "./get_nb_assets.sh -nbmaster <master_server> -username <username> -password <password> -domainname <dname> -domaintype <unixpwd/nt>"
```